sdk/include/dxsdk/axextend.idl

```diff
@@ -16,13 +16,18 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
+#if 0
```

Comment:
Why not just removing it altogether?

Reply:
Because I thought that it's required by new added definitions, and without this, compilation will fail, but, as it appeared, it doesn't fail. So I guess this indeed can be safely removed.
But if it was defined in Wine for some reason, then it is definitely used by something other there, unlike ROS, or is just made for the correct compilation.

Reply:
ok no problem just keep it as it is.

Reply:
The makedep pragma is used by (surprise, surprise) the makedep tool, not the compiler. The compiler may complain about unknown pragmas, makedep will ignore the `#if`, so this is very much correct.

But all that is beside the point. The `#if 0` is there in Wine's version of the file, so if we're doing any sort of syncing we should keep it this way.